src/Enum.php

```diff
@@ -191,6 +213,22 @@ public static function __callStatic($name, $arguments)
         throw new \BadMethodCallException("No static method or enum constant '$name' in class " . \get_called_class());
     }
 
+    /**
+     * Returns a title cased representation of a key
+     *
+     * @param string $key
+     *
+     * @return string
+     */
+    private static function keyToTitleCase($key)
+    {
+        $key_parts = explode('_', $key);
```

Comment:
No need to explode, can just be `ucwords(str_replace('_', ' ', $key))`;

Reply:
ahhh but `ucwords('KEY') // KEY` so we gotta go more `return ucwords(strtolower(str_ireplace('_', ' ', $key)));` but nice improvement!